frontend/app/course/[courseId]/[videoId]/page.tsx

```diff
@@ -54,18 +54,18 @@ export default async function CourseDetailPage({
     )
     const answers: AnswerType[] = await getAnswersData.json()
 
-    return (
-      <CourseDetailWrapper
-        courseId={params.courseId}
-        initialCourseData={initialCourseData}
-        session={session}
-        userId={userId}
-        videoId={params.videoId}
-        questions={questions}
-        answers={answers}
-        questionId={searchParams.questionId}
-      />
-    )
+    let WrapperProps = {
```

Comment:
```suggestion
    const wrapperProps = {
```

再代入しない値はconstで行きましょう！
後冒頭は小文字で。

Reply:
こちらと他の該当箇所も修正しました！